packages/@aws-cdk/toolkit-lib/CODE_REGISTRY.md

```diff
@@ -0,0 +1,31 @@
+## Toolkit Code Registry
+
+| Code | Description | Level | Data Interface |
+|------|-------------|-------|----------------|
+| CDK_TOOLKIT_I1000 | Provides synthesis times. | info | n/a |
+| CDK_TOOLKIT_I1901 | Provides stack data | result | [StackData](docs/interfaces/StackData.html) |
```

Comment:
doc pages aren't committed so i don't know exactly what to do here

Reply:
I think we can link to the public page once it's available.